### DISCUSSION TITLE
How can we ignore the timezone on the DateTime picker ?

### DISCUSSION THREAD

**Discussion by VaZark** [mantinedev/mantine#5423] [Q&A]
I'd like to use the classic `input type="date"` that returns the _YYYY-MM-DD_ with the Mantine's DatePicker for consistent UI across the application.

Currently the _DatePickerInput_ returns the date with timezone information. While this is super useful, it doesn't work for certain types of data where the date is the highest level required

**Comment by rtivital** [comment]
You can convert the returned Date object to a string on your side.

**Reply by sethen** [reply]
> You can convert the returned Date object to a string on your side.

Can you explain what you mean?  I'm trying to format the string coming back and having a tough time when sending it to an API.

**Reply by rtivital** [reply]
```tsx
const format = date => `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`;
```
